src/services/heatmap.service.ts

```diff
@@ -33,7 +33,7 @@ async function getHeatmaps(connection: Connection, startDate: Date, endDate: Dat
                 _id: 0,
                 'dayOfWeek': { $add: [{ $dayOfWeek: "$date" }, -1] },
                 'hour': { $add: ['$arrayIndex', 1] },
-                'interactions': 1,
+                'interactions': { $add: ['$thr_messages', { $arrayElemAt: ['$lone_messages', '$arrayIndex'] }] },
```

Comment:
It should be thr_messages + lone_messages + replier 
Please check discord https://discord.com/channels/915914985140531240/1074268819935203438/1075830031575695390